config/tslint.json

```diff
@@ -24,5 +24,6 @@
 			"allow-leading-underscore",
 			"allow-pascal-case"
 		]
-	}
+	},
+	"jsRules": true
```

Comment:
This inherits the existing typescript rules for javascript as well, rather than specifying the same set twice